### PR TITLE
Text utilities   list

### PR DESCRIPTION
**Pull request by anandkumarcoder**
Feat: Text utilities and list

image
image
image
image
image
image

### CODE REVIEW

File: docs/docs.css [hunk 1]
```diff
@@ -115,6 +116,19 @@ p {
...
   flex-direction: column;
 }
+.component-container-text-utility {
+  flex-direction: column;
+  align-items: center;
+}
+
+.text-alignment{
+/* align-items: initial; */
+display: block;
```
Comment (ravikumar0403) [inline docs/docs.css:126]: You can remove commented code if not required

File: docs/docs.css [hunk 2]
```diff
@@ -166,18 +180,29 @@ footer {
...
   color: var(--primary-font-color);
 }
 
-li {
+/* li {
   list-style: none;
-  display: inline-block;
-}
+   display: inline-block; 
+} */
```
Comment (ravikumar0403) [inline docs/docs.css:208]: This part can also be removed if you don't need it

**Review by ravikumar0403 — APPROVED** [review]
Looks good. left a couple of sanity comments

**Review by chanchal16 — COMMENTED** [review]
Looks good to me, just one suggestion remove the commented code.

**Pull request merged**